cime_config/config_pes.xml

```diff
@@ -602,4 +602,42 @@
     </mach>
   </grid>
 
+  <grid name="a%TL319.+oi%tnx1v4">
```

Comment:
The configuration above (line 153) seems to be fine for fram, whereas on betzy your configuration has less idle processors. But I would suggest to move this configuration up, below the one in line 153. Then it might make sense to specify mach="betzy" here, and maybe compset="_DATM.*_BLOM%ECO" to avoid that this configuration is picked unintendely instead of the one in line 415

Reply:
Thanks for input, I will set mach="betzy" and compset="_DATM.*_BLOM"
I would like to keep the option to run with and without iHAMOCC. Once in a while it is interesting to check how much extra resources are needed for running the model with BGC.